calendar/quickstart/quickstart.py

```diff
@@ -30,7 +30,7 @@ def main():
     # created automatically when the authorization flow completes for the first
     # time.
     store = file.Storage('token.json')
-    creds = store.get()
+    creds = None
```

Comment:
This seems accidental. Please revert.

Reply:
this snippet wouldn't work on python 3.7 unless I made this change